libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -151,54 +147,44 @@ public void onFailure(Call call, IOException exception) {
 
   @Override
   public void onResponse(Call call, Response response) throws IOException {
-    List<String> revokedKeys = extractResponse(response);
-
-    saveBlackList(revokedKeys);
+    saveBlackList(response);
   }
 
-  private String createListContent(List<String> revokedKeys) {
-    Date date = new Date();
+  private JsonObject addSHAtoKeys(JsonObject jsonObject) {
+    Gson gson = new Gson();
 
-    StringBuilder content = new StringBuilder(date.getTime() + NEW_LINE);
+    JsonArray jsonArray = jsonObject.getAsJsonArray("RevokedCertKeys");
+    JsonArray shaKeys = new JsonArray();
 
-    for (String key : revokedKeys) {
-      content.append(SHA256).append(key).append(NEW_LINE);
+    for (JsonElement key : jsonArray) {
+      shaKeys.add(SHA256 + key.getAsString());
```

Comment:
why are we doing this? is this how server wants them to be formatted and then why blacklisted hashes aren't sent to us in the right format?

Reply:
Correct on both accounts. This is how the hashes need to be formatted and they are sent to us without this formatting. We had a discussion about it at the time, but I do not recall the reason this wasn't done. We can make a request to the backend team to get this change made to make everything easier on us.